packages/augur-sdk/src/utils.ts

```diff
@@ -131,15 +137,23 @@ export function calculatePayoutNumeratorsValue(
       .dividedBy(new BigNumber(numTicks, 10))
       .plus(new BigNumber(displayMinPrice, 10))
       .toString();
+  } else {
+    if (isInvalidCategorical(payout)) { // or yes/no
+      return MALFORMED_OUTCOME;
+    }
+
+    return payout.findIndex((item: string) => Number(item) > 0).toString();
   }
-  // test if stake payout is malformed
-  if (
-    payout.reduce((p, ticks) => (parseInt(ticks, 10) > 0 ? p + 1 : p), 0) > 1
-  ) {
-    return MALFORMED_OUTCOME;
-  }
+}
+
+function isInvalidCategorical(payout: string[]): boolean {
```

Comment:
this doesn't make sense. 

isInvalidCategorical but is testing for malformed payout numerators. 

Invalid and malformed are different concepts that don't mix

Reply:
yikes fixed